### PR TITLE
Weather - Properly sync overcast value

### PR DESCRIPTION
**Pull request by ulteq**
* Makes sure that all clients share the same (numeric) overcast value
* Only affects the barometric pressure calculation
* Does not affect the actual weather on the clients

Avoids barometric pressure synchronization issues between clients due to overcast drift.

Reference: https://cuel.github.io/

### CODE REVIEW

File: addons/weather/XEH_postInit.sqf [hunk 1]
```diff
@@ -11,13 +11,14 @@ GVAR(wind_period_start_time) = CBA_missionTime;
 if (!isServer) then {
     "ACE_MISC_PARAMS" addPublicVariableEventHandler {
         TRACE_1("MISC PARAMS PVEH",ACE_MISC_PARAMS);
+        GVAR(currentOvercast) = (ACE_MISC_PARAMS select 0);
```
Comment (jonpas) [inline addons/weather/XEH_postInit.sqf]: I don't know why there are `()` on the right side in the rest of the params, but they are really pointless.

Comment (ulteq) [reply addons/weather/XEH_postInit.sqf]: We can fix it in the following cleanup PRs.

Comment (PabstMirror) [reply addons/weather/XEH_postInit.sqf]: This only runs on non server's
So GVAR(currentOvercast) is always 0 on server?

**Comment by PabstMirror** [comment]
imho, instead of syncing with a pub variable "event" in ACE_MISC_PARAMS 
we should just do `missionNamespace setVariable ["ace_weather_overcast", overcast, true]`
skip the need for any sqf to run on the clients

**Comment by ulteq** [comment]
Yes, I'm going to remove those public event handlers entirely in the following cleanup pull request.

File: addons/weather/XEH_postInit.sqf (hunk 1 above)
Comment (ulteq) [reply addons/weather/XEH_postInit.sqf]: The server never uses:
```GVAR(currentTemperature), GVAR(currentHumidity) and GVAR(currentOvercast)```

**Pull request merged**